### ISSUE TITLE
Image profile needs to permit text equivalent

### ISSUE THREAD

**Issue by plehegar** [w3c/imsc#6]
IMSC1 Image profile [1] defined in chapter 6 does not permit a text equivalent. If this profile is used to provide captions as an alternative to audio then it will fail the WCAG2 guideline 1.2 [2]. From a practical perspective this would mean that users who can neither see nor hear well can not use tools to translate the content into e.g. Braille.

[1] http://www.w3.org/TR/ttml-imsc1/#image-profile-constraints
[2] http://www.w3.org/TR/2008/REC-WCAG20-20081211/#media-equiv

Some potential ways to solve this:
1. Permit text to be included within the image profile that describes the images.
2. Require or recommend that when image profile captions are provided there should be a text profile caption track available simultaneously.

(raised by Nigel Megitt on 2014-05-21)
From tracker issue http://www.w3.org/AudioVideo/TT/tracker/issues/309
